src/main/webapp/css/main.css

```diff
@@ -14,8 +14,20 @@
  * limitations under the License.
  */
 
+/*
+  Setting our title theme */
+title {
+    font-family: Futura-PT;
```

Comment:
When it comes to fonts in CSS, you either need to use a font that is common/generic enough that all browsers support it, include code that will download the font if it is not installed, or specify fall back fonts that should be used if the font isn't available (or a combination of the above ... code to download plus a fallback font).

As Futura PT appears to be a proprietary Adobe font, it is unlikely that this will work on any computer except for ones that have the Adobe Creative Suite installed and that have installed this particular font (unless you make the font available for download on the server and include an appropriate `<link>` tag to download it, something which you should not do when it comes to using a public, open source repository such as this one).

To use a fallback, you can do something like:

     font-family: Futura-PT, sans-serif;

... which essentially uses a generic sans-serif font if Futura-PT is not available.

If you're looking at customizing font, you may want to take a look at [Google Fonts](https://fonts.google.com); not only are those fonts free to use, but the website will provide you with the necessary `<link>` tag that you need to add in the `<head>` section of the corresponding `<html>` page in order to cause the font to be downloaded.